### DISCUSSION TITLE
关于在记事中添加图片后，相册内出现相同图片的问题

### DISCUSSION THREAD

**Discussion by Loueries** [WatsonYao/nian_app#65] [Q&A] [answered]
首先感谢开发者大大们做出如此功能齐全的APP！使用时间比较短只有一年左右，但在使用过程中遇到了很困扰的问题：
从新手指南中得知，此相同图片重复的原因大概是将发布的图片等轻量级数据放到nian文件夹存储起来，但是清理相册或者在浏览的时候经常会出现两张相同照片误删其中一张导致记事内图片缺失的情况...不知能通过什么方式回避重复照片出现的情况，所以来讨论组求助一下QAQ

**Comment by celestialpivot** [comment]
如果只是不想在相册里重复显示，你可以看一下你手机有没有隐藏相册or屏蔽相册的功能，因为nian存储相册用的是专门的文件夹，你只要把你的nian相册隐藏掉不在你的图库里显示就好

**Comment by WatsonYao** [comment, marked as answer]
感谢使用和反馈，清理工具一般都提供了白名单功能，白名单下的文件一般不会被清理，将 nian 文件夹添加到白名单即可。
